test/gtest/db_sync.cpp

```diff
@@ -44,9 +44,13 @@
 #include <exception>
 #include <unordered_set>
 
-#define WORKAROUND_ISSUE_2493 1
+/// \todo HACK
+/// This should be set to 1 if either WORKAROUND_ISSUE_2492_GRANULARITY_LOSS
+/// or WORKAROUND_ISSUE_2492_TINY_TENSOR is defined as non-zero in
+/// src/solver/conv_winoRxS.cpp
+#define WORKAROUND_ISSUE_2492 1
 
-#if WORKAROUND_ISSUE_2493 && defined(_WIN32)
+#if WORKAROUND_ISSUE_2492 && defined(_WIN32)
```

Comment:
The original W/A (#2507, #2510) has a typo (2492->2493) which is fixed here at once.